[core]/es_extended/server/commands.lua

```diff
@@ -35,41 +35,45 @@ local upgrades = Config.SpawnVehMaxUpgrades and
     } or {}
 
 ESX.RegisterCommand('car', 'admin', function(xPlayer, args, showError)
-	local playerPed = GetPlayerPed(xPlayer.source)
-	local playerCoords = GetEntityCoords(playerPed)
-	local playerHeading = GetEntityHeading(playerPed)
-    local playerVehicle = GetVehiclePedIsIn(playerPed)
-
-	if not args.car or type(args.car) ~= 'string' then
-		args.car = 'adder'
-	end
-
-	if playerVehicle then
-		DeleteEntity(playerVehicle)
-	end
+	if xPlayer then
```

Comment:
reverse logic is better and clean

if not xPlayer then
  return notify....
end